server/src-lib/Hasura/GraphQL/NormalForm.hs

```diff
@@ -266,8 +266,8 @@ data AnnGEnumValue
   | AGEReference !RQL.EnumReference !(Maybe RQL.EnumValue)
   deriving (Show, Eq)
 
-data AnnGValue
-  = AGScalar !PGScalarType !(Maybe PGScalarValue)
+data AnnGValue (b :: Backend)
```

Comment:
We might as well delete this module if it is not being used anywhere?

Reply:
Done.